Source/WebKit/Shared/PALArgumentCoders.serialization.in

```diff
@@ -20,8 +20,8 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-header: <pal/graphics/WebGPU/WebGPUTextureFormat.h>
-enum class PAL::WebGPU::TextureFormat : uint8_t {
+header: <WebCore/WebGPUTextureFormat.h>
```

Comment:
maybe in subsequent patches this file is renamed..

Reply:
Oh, good point, this shouldn't be in PALArgumentCoders any more.